lib/modules/manager/flux/extract.ts

```diff
@@ -32,8 +33,9 @@ function readManifest(content: string, file: string): FluxManifest | null {
   const manifest: FluxManifest = {
     kind: 'resource',
     file,
-    releases: [],
-    repositories: [],
+    helmReleases: [],
```

Comment:
please don't rename if not required, or do separate refactor PR.

Reply:
Any chance for an exception here? repositories can be many things in Flux.

Reply:
I will open a refactor PR.